src/ProductlineApp.WebUI/ClientApp/src/components/atoms/buttons/auctionPortalsButtons/AllegroAuctionsButton.tsx

```diff
@@ -0,0 +1,25 @@
+import AllegroIcon from '../../../../assets/icons/allegro_a_logo_icon.svg';
+import './css/allegroAuctionsButton.css';
+import { PlatformEnum } from '../../../../enums/platform.enum';
+
+function AllegroAuctionsButton({
+  selectedAuctionPortal,
+  handleClickTypeAuctionPortalButton,
+  id,
+}: {
+  selectedAuctionPortal: PlatformEnum;
```

Comment:
to jest potencjalnie undefined, wiec musi byc "selectedAuctionPortal?: PlatformEnum;"